src/views/tickets/orders-rsvp.php

```diff
@@ -26,7 +26,7 @@
 $attendee_groups = $view->get_event_rsvp_attendees_by_purchaser( $post_id, $user_id );
 ?>
 <div class="tribe-rsvp">
-	<h2><?php printf( esc_html__( 'My RSVPs for This %s', 'event-tickets' ), $post_type->labels->singular_name ); ?></h2>
+	<h2><?php printf( esc_html__( 'My %1$s for this %2$s', 'event-tickets' ), tribe_get_rsvp_label_plural( basename( __FILE__ ) ), $post_type->labels->singular_name ); ?></h2>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$post_type'

<sub>hash: 47267499e8dfa5bf431f6733e38538ef</sub>